bindgen/codegen/mod.rs

```diff
@@ -4148,30 +4073,30 @@ impl TryToRustTy for FunctionSig {
             ClangAbi::Known(Abi::ThisCall)
                 if !ctx.options().rust_features().thiscall_abi =>
             {
-                warn!("Skipping function with thiscall ABI that isn't supported by the configured Rust target");
-                Ok(proc_macro2::TokenStream::new())
```

Comment:
Does this mean that now we stop generating code at all if we don't support the ABI?

Reply:
Probably. I think not generating code at all is more correct than generating an empty type though.

Reply:
Then this needs to go in the changelog. I'm surprised none of the tests caught this.

Reply:
Well, the tests did not trigger because the output should still be the same, right?

Previously, it would output an empty token stream, now it outputs nothing.

Reply:
Yeah, we only stop generating code for individual items, not  completely.